### PR TITLE
Align with JS counterpart

### LINKED ISSUE #3: Align with JS Counterpart
We have the C# repository client library sample project finished by this time we want it to align with its JS counterpart. Mostly importantly, the code should be very similar to the JS version, so is other things such as documentation and pipeline.

### PR DESCRIPTION
**Pull request by lf-jason-jia**
Fix #3 
For details, see the linked issue.

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,38 +1,17 @@
...
+            // Get credentials
+            var config = new ServiceConfig("TestConfig.env");
 
             // Create the client
-            var repoClient = RepositoryApiClient.Create(servicePrincipalKey, accessKey);
+            var repoClient = RepositoryApiClient.Create(config.ServicePrincipalKey, config.AccessKey);
 
             var rootEntryId = 1;
-            var entryListing = await repoClient.EntriesClient.GetEntryListingAsync(repositoryId, rootEntryId);
+            var entryListing = await repoClient.EntriesClient.GetEntryListingAsync(config.RepositoryId, rootEntryId);
```
Comment (steven-xu-lf) [inline Program.cs]: the program in the js sample app also prints out the repo name, root folder along with its root folder children, should it also be done in the C# counter part sample project?

**Review by steven-xu-lf — COMMENTED** [review]
im not sure if this is nessesary or not for both sample projects, but does the .csproj for this sample app need to use the most recent version of the repo api client since i noticed that it still uses the 1.0.1-beta.1 version, while version 1.0.1 is now published on nuget but im not sure if they are the same tho,

File: Program.cs (hunk 1 above)
Comment (lf-jason-jia) [reply Program.cs]: Sorry I only committed the change but forgot to push it. It's updated now.

**Comment by lf-jason-jia** [comment]
> 

Yes, I will update it.

**Review by steven-xu-lf — APPROVED** [review]

**Pull request merged**